c-pallets/file-map/src/lib.rs

```diff
@@ -113,6 +120,10 @@ pub mod pallet {
 	#[pallet::storage]
 	#[pallet::getter(fn scheduler_map)]
 	pub(super) type SchedulerMap<T: Config> =
+	// StringLimit is set as the max numbers of SchedulerInfo objects can exist on storage
+	// It seems wrong? StringLimit is related to SchedulerInfo.ip and other parameters
+	// max length where the type is BoundedVec<u8, StringLimit>
+	// You should set another constant for this limit, not reuse StringLimit
```

Comment:
Your proposal is correct. I also have this plan, but I haven't determined the upper limit. Now we plan to suspend the upper limit to `10000`:
https://github.com/CESSProject/cess/pull/88/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR976